msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -427,6 +427,79 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+> [!NOTE]
+>
+> * Currently, generate a deep link to share content to stage in meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * Deep link to share content to stage in meeting is supported in Teams desktop client only.
+
+When a deep link is selected in an app by a user who is part of an ongoing meeting, then the app is shared to the stage and a permission pop-up window appears. Users can grant permissions for the participants, such as co-editing a document or collaborating with an app.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/screenshot-of-pop-up-permission.png" alt-text="The screenshot is an example that shows a permission pop-up window.":::
```

Comment:
Can we update this image to what it would look like for a third-party app? (ie: not Word, which specifically uses the "co-edit" language in the dialog)
For example:
![image](https://user-images.githubusercontent.com/83943/193703505-dcece295-c195-42bb-b36c-6e76a1a39fe5.png)

Reply:
Updated as suggested